nncf/quantization/algorithms/min_max/openvino_backend.py

```diff
@@ -181,7 +181,7 @@ def get_statistic_collector(
                     f"Aggregator type: {params.aggregator_type} is not supported for OpenVino PTQ backend yet."
                 )
 
-            kwargs = {"reduction_shape": reduction_shape, "inplace": inplace}
+            kwargs = {"reduction_axes": reduction_shape, "inplace": inplace}
```

Comment:
Why the `reduction_shape` wasn't changed?

Reply:
Did it, thanks